contents/apps/launchdarkly/docs.md

```diff
@@ -0,0 +1,39 @@
+---
+title: LaunchDarkly Connector documentation
+showTitle: true
+topics:
+    - launchdarkly
+---
+
+### What does the LaunchDarkly Connector do?
+This app exports actions — such as toggling a feature flag —  from LaunchDarkly and into PostHog via the API. This is useful for analyzing LaunchDarkly events using PostHog's product analytics suite and alongside other product data.
+
+###### What are the requirements for this app?
+Using the LaunchDarkly connector requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later. 
+
+Not running 1.30.0? Find out [how to update your self-hosted PostHog deployment](https://posthog.com/docs/self-host/configure/upgrading-posthog)!
+
+You'll also need a LaunchDarkly account, obviously.
+
+##### How do I get started with the LaunchDarkly Connector?
+INFO COMING HERE
+
+#### How are events imported to PostHog?
+INFO COMING HERE
```

Comment:
Events from LaunchDarkly are automatically sent to the PostHog API and will have the following schema:

{
  "event": "string",
  "properties": {
    "launchDarklyEventId": "string",
    "launchDarklyEventAccountId": "string",
    "launchDarklyEventType": "string",
    "comment": "string",
    "distinct_id": "string",
    "email": "string",
    "firstName": "string",
    "lastName": "string",
    "title": "string"
  },
  "timestamp": "rfc3339 timestamp"
}